app/src/screens/Call/hooks.ts

```diff
@@ -0,0 +1,227 @@
+import { AudioDevice as TwilioAudioDevice } from '@twilio/voice-react-native-sdk';
+import React from 'react';
+import { useTypedDispatch } from '../../store/app';
+import {
+  type ActiveCall,
+  disconnectActiveCall,
+  muteActiveCall,
+  sendDtmfActiveCall,
+} from '../../store/voice/call/activeCall';
+import {
+  type AudioDevicesState,
+  getAudioDevices,
+  selectAudioDevice,
+} from '../../store/voice/audioDevices';
+import {
+  useActiveCall,
+  useActiveCallRemoteParticipant,
+  useActiveCallStatus,
+} from '../../hooks/activeCall';
+import { useAudioDevices } from '../../hooks/audioDevices';
+
+/**
+ * Hook for dialpad handlers and state.
+ * @param activeCall - The active call.
+ * @param dispatch - The dispatch function for the Redux store.
+ * @returns - Handlers and state for the dialpad.
+ */
+const useCallDialpad = (
+  activeCall: ActiveCall,
+  dispatch: ReturnType<typeof useTypedDispatch>,
+) => {
+  const [isVisible, setIsVisible] = React.useState<boolean>(false);
+
+  const [handle, isDisabled] = React.useMemo(() => {
+    const noOp = () => {};
```

Comment:
`noop` function is duplicated multiple times in this file. What do you think of putting this at the top and reusing it?